reports/api/request.go

```diff
@@ -41,6 +42,11 @@ func (req addReportConfigReq) validate() error {
 	if err := req.Schedule.Validate(); err != nil {
 		return errors.Wrap(err, apiutil.ErrValidation)
 	}
+	if req.ReportTemplate.String() != "" {
+		if err := req.ReportTemplate.Validate(); err != nil {
+			return errors.Wrap(err, apiutil.ErrValidation)
+		}
+	}
```

Comment:
If template is not available then it will use default template `MG_REPORTS_TEMPLATE` ?